apps/api-extractor/package.json

```
@@ -58,6 +57,10 @@
     "@types/lodash": "4.14.116",
     "@types/node": "12.20.24",
     "@types/resolve": "1.20.2",
-    "@types/semver": "7.3.5"
+    "@types/semver": "7.3.5",
+    "typescript": "~4.8.4"
+  },
+  "peerDependencies": {
+    "typescript": "~4.8.4"
```

```suggestion
    "typescript": "^4.8.4 || ^5.0.0"
```

I think this is the right SemVer range to accommodate TypeScript 4.9 and 5.0. You can check it with [npm SemVer Calculator](https://semver.npmjs.com/).

---

Since api-extractor has moved to TS v5, I have updated the peer deps version to only include v5 and not v4